使用Linux编译android的paddle-mobile库时报错

在执行`sh build.sh android`命令时报错，错误信息如下：
```
test@test:~/android_linux/paddle-mobile/tools$ sh build.sh android
CMake Error at CMakeLists.txt:2 (project):
  The CMAKE_C_COMPILER:

    /home/test/android_linux/android-ndk-r17b/toolchains/llvm/prebuilt/linux-x86_64/bin/clang

  is not a full path to an existing compiler tool.

  Tell CMake where to find the compiler by setting either the environment
  variable "CC" or the CMake cache entry CMAKE_C_COMPILER to the full path to
  the compiler, or to the compiler name if it is in the PATH.


CMake Error at CMakeLists.txt:2 (project):
  The CMAKE_CXX_COMPILER:

    /home/test/android_linux/android-ndk-r17b/toolchains/llvm/prebuilt/linux-x86_64/bin/clang++

  is not a full path to an existing compiler tool.

  Tell CMake where to find the compiler by setting either the environment
  variable "CXX" or the CMake cache entry CMAKE_CXX_COMPILER to the full path
  to the compiler, or to the compiler name if it is in the PATH.


-- Configuring incomplete, errors occurred!
See also "/home/test/android_linux/paddle-mobile/build/release/arm-v7a/CMakeFiles/CMakeOutput.log".
```

--- a/doc/development_doc.md
+++ b/doc/development_doc.md
@@ -143,11 +143,16 @@ root@5affd29d4fc5:/ # make
 ##### 下载Android NDK
 
 从源码交叉编译paddle-mobile,用户需要提前准备好交叉编译环境。Android平台使用的C/C++交叉编译工具链是[Android NDK](https://developer.android.com/ndk/)，用户可以自行前往下载，也可以通过以下命令获取：
-
+- Mac平台
 ```
 wget https://dl.google.com/android/repository/android-ndk-r17b-darwin-x86_64.zip
 unzip android-ndk-r17b-darwin-x86_64.zip
 
+```
+- Linux平台
+```
+wget https://dl.google.com/android/repository/android-ndk-r17b-linux-x86_64.zip
+unzip android-ndk-r17b-linux-x86_64.zip
 ```
 
 ##### 设置环境变量

